### ISSUE TITLE
Dependency Incompaitbity

### ISSUE THREAD

**Issue by omerasif57** [Baseflow/flutter_cache_manager#292]
## 💬 Questions and Help

I have two dependecies
```
flutter_cache_manager: ^2.1.1
flutter_native_splash: ^1.0.3
```

### Pub get Error ###

```
Running "flutter pub get" in flutter_app...                     
Because flutter_native_splash >=1.0.1 depends on image ^3.0.1 and flutter_cache_manager 2.1.1 depends on image ^2.1.18, flutter_native_splash >=1.0.1 is incompatible with flutter_cache_manager 2.1.1.

And because no versions of flutter_cache_manager match >2.1.1 <3.0.0, flutter_native_splash >=1.0.1 is incompatible with flutter_cache_manager ^2.1.1.

So, because flutter_app depends on both flutter_cache_manager ^2.1.1 and flutter_native_splash ^1.0.3, version solving failed.
pub get failed (1; So, because flutter_app depends on both flutter_cache_manager ^2.1.1 and flutter_native_splash ^1.0.3, version solving failed.)
exit code 1
```

Please advise on how can I fix this issue.

**Comment by peterbyCom** [comment]
same.

**Comment by omerasif57** [comment]
using `flutter_native_splash: ^0.3.2` working fine for me.

**Comment by renefloor** [comment]
I'd guess this is fixed in `flutter_cache_manager: ^3.0.0`, otherwise please reopen the issue.
